SwrveSDKCommon/src/main/java/com/swrve/sdk/ISwrveCommon.java

```diff
@@ -0,0 +1,52 @@
+package com.swrve.sdk;
+
+import android.content.Context;
+
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Map;
+
+public interface ISwrveCommon
+{
+    String getApiKey();
+
+    String getSessionKey();
+
+    short getDeviceId();
+
+    int getAppId();
+
+    String getUserId();
+
+    String getAppVersion();
+
+    String getUniqueKey();
+
+    String getBatchURL();
+
+    String getCachedLocationData();
```

Comment:
You still want this, or did we talk ourselves out of it yesterday?

If we do this, ISwrveLocationSDK would live, I'm assuming, in the SwrveLocationSDK, but then SwrveSDK real would need to implement it, would it not?  In that case, SwrveSDK proper would need to rely on SwrveLocationSDK, which I don't think we really want.